src/julia-parser.scm

```diff
@@ -54,9 +58,25 @@
            (lambda (x)
              (has? t x))))))
 
+; only allow/strip suffixes for some operators
+(define no-suffix? (Set (append prec-assignment prec-conditional prec-lazy-or prec-lazy-and
```

Comment:
It was a bit easier as a blacklist because otherwise I'd have to split e.g. `prec-arrow ` into a couple of separate lists rather than just explicitly listing `-- --> ->` here.